deep4cast.py

```diff
@@ -21,72 +23,133 @@ def main(args):
     df = read_table(args.data_path, sep=',')
     ts = df.values
 
+    # Format data to shape (n_steps, n_vars, n_series)
+    while len(ts.shape) < 3:
+        ts = np.expand_dims(ts, axis=-1)
+
     # Prepare train and test set. Make sure to catch the case when the user
     # did not supply a test. We use the end of the time series for testing
     # because of lookahead bias.
     if args.test_fraction:
         test_length = int(len(df) * args.test_fraction)
         train_length = len(df) - test_length
         ts_train = ts[:-test_length]
-        ts_test = ts[-test_length - args.lookback_period:]
+        ts_test = ts[-test_length - args.lag:]
     else:
         ts_train = ts
 
-
-    # Why isn't topology an option?
```

Comment:
This comment still holds

Reply:
I see. Well, now we do have one forecaster class :-). What we could do is have some default topologies like you mentioned in your comment. I think I didn't understand the comment correctly at first. We can certainly do that.